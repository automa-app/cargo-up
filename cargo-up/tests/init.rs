use insta::assert_snapshot;
mod utils;

#[test]
fn test_init() {
    let (out, err) = utils::run_upgrader("init", "0.5.0", false);
    assert_snapshot!(out);
    assert_snapshot!(err);
}
