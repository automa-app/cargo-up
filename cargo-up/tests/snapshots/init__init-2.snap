---
source: tests/init.rs
expression: err
---


Can't upgrade from 0.4.0

